### DISCUSSION TITLE
SIS graphics card

### DISCUSSION THREAD

**Discussion by TechGuyC** [86Box/86Box#3709] [Ideas]
Add SIS 6326 AGP 

SIS 6326 AGP doucmentation:
[SIS 6326.pdf](https://github.com/86Box/86Box/files/12765629/SIS.6326.pdf)

SIS 6326 AGP bios
[OEMROM.zip](https://github.com/86Box/86Box/files/12765726/OEMROM.zip)

**Comment by jriwanek** [comment]
That documentation is basically the user manual, and completely worthless for implementing emulation of the chip.

**Comment by TechGuyC** [comment]
> That documentation is basically the user manual 


i think you did not read correctly the manual and did not understand how to add the chipset emulation

>and completely worthless for implementing emulation of the chip.

By the way you said "worthless",it sounded like the documentation did not say how is the system of the chip

**Reply by jriwanek** [reply]
It literally says nothing useful on the chipset implementation, opening more issues making the same request is btw *spam* and if you continue you'll be barred from opening others in the future

**Reply by TechGuyC** [reply]
i found this other documentation,this should work
[6326 documentation.pdf](https://github.com/86Box/86Box/files/13326339/6326.documentation.pdf)
